test/remote/remote-app.spec.ts

```diff
@@ -0,0 +1,78 @@
+import {AutomationEnvironment, Browser, BrowserType, RemoteApplication, RemoteApplicationManager} from '../../src';
+import {ComponentsPage} from '../pages/components.po';
+
+describe('Remote application test', () => {
+  // Manually start the remote-application-server ( npm run start-remote-test-server) in a separate terminal
```

Comment:
This could be removed if added to previous README.md

Reply:
Removed